engine-tests/src/main/java/org/terasology/engine/testUtil/WithUnittestModule.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.engine.testUtil;
+
+import org.terasology.engine.context.Context;
+import org.terasology.engine.core.GameEngine;
+import org.terasology.engine.core.module.ModuleManager;
+import org.terasology.engine.core.subsystem.EngineSubsystem;
+import org.terasology.gestalt.module.Module;
+
+public class WithUnittestModule implements EngineSubsystem {
```

Comment:
any suggestions for the name? should it end it `Subsystem`?

Reply:
A survey of what we currently (including this PR) have shows:

- Usage in extends/implements clause: 22
- 18 with `Subsystem` suffix / 5 without suffix
- classes without suffix are
	- WithUnittestModule
	- OpenVRInput 
	- Headless[Audio|Graphics|Input]
	
Given that, I'd suggest to
- add the `Subsystem` suffix here. Would `RegisterUnittestModuleSubsystem` be a reasonable name?
- (different PR) rename `OpenVRInput` >>> `OpenVrInputSubsystem`
- (different PR) add `Subsystem` suffix to `Headless[Audio|Graphics|Input]`
- (different PR) make sure to consistently use `Subsystem` (we use `SubSystem` in some places)

The latter changes would fit the topic of code style improvements @jdrueckert :wink:

Reply:
I'd appreciate a short docstring comment for this subsystem explaining why we have it, and what it is used for.